dev/src/write-batch.ts

```diff
@@ -162,6 +164,13 @@ export class WriteBatch {
     return this._ops.length === 0;
   }
 
+  /**
+   * @private
+   */
+  get opCount(): number {
```

Comment:
Is the opCount is used for anything other than an "index" or "key" of a `WriteBatch` in `BulkCommitBatch.resultsMap`?  I was kind of confused when trying to understand its usage in `BulkCommitBatch` because if the opCount is just a convenient monotonically-increasing ID then I'd recommend generating your own ID in BulkCommitBatch to improve readability.

Reply:
Good point! I originally exposed `opCount` in `WriteBatch` to avoid having an extra field on `BulkCommitBatch`, but I didn't consider readability.